src/content/docs/fr/guides/dev-toolbar.mdx

```diff
@@ -34,7 +34,7 @@ La barre d'outils de développement vise à fournir un moyen rapide et facile de
 
 ### Settings
 
-L'application Settings vous permet d'activer différents paramètres pour la barre d'outils de développement, tels que la journalisation verbeuse et la possibilité de désactiver les notifications.
+L'application Paramètres vous permet de configurer les options de la barre d'outils de développement, telles que la journalisation verbeuse, la désactivation des notifications et l'ajustement de son emplacement sur l'écran.
```

Comment:
```suggestion
L'application Settings vous permet de configurer les options de la barre d'outils de développement, telles que la journalisation verbeuse, la désactivation des notifications et l'ajustement de son emplacement sur l'écran.
```

The title of this section is `Settings`, I think we should keep `Settings` in this sentence too, or change all the page titles to French.

Reply:
I wrote it correctly last time without translating it, but now I've translated it all into French. Thanks for the review! 🙌